samples/display/cfb_shell/boards/frdm_k64f.conf

```diff
@@ -0,0 +1,2 @@
+CONFIG_I2C=y
```

Comment:
Remove this file, it should not be there.

Reply:
ok